app/src/main/java/tech/relaycorp/gateway/common/Keys.kt

```diff
@@ -0,0 +1,8 @@
+package tech.relaycorp.gateway.common
+
+import tech.relaycorp.relaynet.wrappers.deserializeRSAKeyPair
+import java.security.PrivateKey
+import java.security.PublicKey
+
+fun PrivateKey.toPublicKey(): PublicKey =
+    encoded.deserializeRSAKeyPair().public
```

Comment:
Didn't we define a function somewhere to do this without serialising + deserialising? I'm pretty sure we we discussed it on Slack a few weeks ago. I can't find it though 🤔

Reply:
Found it, it was for the `privateAddress` method. But we'll need to import BounceCastle now, because we didn't expose that internal code.